modules/core/src/main/java/org/apache/ignite/internal/lang/IgniteInternalCheckedException.java

```diff
@@ -50,7 +50,12 @@ public class IgniteInternalCheckedException extends Exception implements Traceab
      */
     private final int code;
 
-    /** Unique identifier of this exception that should help locating the error message in a log file. */
+    /**
+     * Unique identifier of the exception that helps locate the error message in a log file.
+     *
+     * <p>Not {@code final} because it gets accessed via reflection when creating a copy.
+     */
+    @SuppressWarnings({"NonFinalFieldOfException", "FieldMayBeFinal"})
```

Comment:
As far as I know, you can change the final field through reflection, please explain in more detail why it is not final.

Reply:
I don't remember, I copied this comment from the other exception. Probably the author of the copying code didn't want to bother with handling `final` (and I agree with him)

Reply:
I suggest either not changing the code, or finishing it somehow. For example, handle the case of the final field or leave a comment with more details.

Reply:
What details do you want to see in the comment? It's literally how this code works now

Reply:
Through reflection we can change the final field, I don’t understand why we shouldn’t use this.

Reply:
The reflection code was not written by me, also I, personally consider changing final fields a hack, and we should avoid it

Reply:
If you don't want to fix it, then let's roll back the change.

Reply:
we have the same comment in the similar exception and it was alright, what the fuck. Whatever, I'll roll back the change, I'm so tired of this arguing